apps/web/package.json

```diff
@@ -130,6 +131,7 @@
     "@types/semver": "^7.3.10",
     "@typescript-eslint/eslint-plugin": "^7.6.0",
     "@typescript-eslint/parser": "^8.18.1",
+    "cheerio": "^1.0.0",
```

Comment:
If we don't want this dependency we could also try simple regex to match script tags but I think it would be more brittle than an actual html parser.